platform/nimiq/tests/getTransactionsByAddress_50.json

```diff
@@ -0,0 +1,422 @@
+[
```

Comment:
why do you use a file? can you use only the necessary cases and put inside the test file, please?

Reply:
Files are independent entities, we should keep it this well for code organization. 

Code should not be mixed with JSON data